libraries/ui-strings/src/main/res/values/localazy.xml

```diff
@@ -130,6 +130,7 @@
   <string name="common_call_invite">"Call in progress (unsupported)"</string>
   <string name="common_call_started">"Call started"</string>
   <string name="common_chat_backup">"Chat backup"</string>
+  <string name="common_copied_to_clipboard">"Copied to clipboard"</string>
```

Comment:
I have added the string to Localazy at https://localazy.com/p/element/source-language/_a7211042194211892347. (there is no action required from you since the key is the same).